lesson4/backend28.js

```diff
@@ -3,9 +3,11 @@ const express= require('express')
 const app = express();
 app.listen(3618);
 app.use(express.static('public'));
+const bodyParser = require('body-parser');
+app.use(bodyParser.json());
 
-app.get('/send',(req,res)=>{
-  const firstdata = req.query.userNames;
-  const seconddata = req.query.userComments;
+app.post('/send',(req,res)=>{
+  const firstdata = req.body.name;
+  const seconddata = req.body.comment;
```

Comment:
I would change `seconddata` to `secondData` to make it camel case. If you do that, change all `seconddata` to camel case.

Reply:
done, Thanks! :)